src/styles/blocks/header.scss

```diff
@@ -0,0 +1,104 @@
+body,
+h1,
+ul,
+li,
+p,
+a {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
You should avoid using global styles in a block file. It's better to move these styles to a separate general file. This way, the styles are more modular and easier to manage.